up_client_socket/rust/utransport-socket/Cargo.toml

```diff
@@ -0,0 +1,27 @@
+[package]
+name = "utransport-socket"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+anyhow = "1.0.75"
+async-std = "1.12.0"
```

Comment:
Seems like you removed usage of `async-std`, so you can remove it as a dependency.